### PR TITLE
move browserid_url to Initializer and add generator

### PR DESCRIPTION
**Pull request by ccarruitero**

### CODE REVIEW

File: devise_browserid_authenticatable.gemspec [hunk 1]
```diff
@@ -12,5 +12,7 @@ Gem::Specification.new do |s|
 
   s.version       = "1.2.0"
   
-  s.add_dependency "devise", "~> 2.1"
+  s.add_dependency 'devise', '3.0.0.rc'
```
Comment (skyschub) [inline devise_browserid_authenticatable.gemspec]: An unstable, not released version? why?

Comment (ccarruitero) [inline devise_browserid_authenticatable.gemspec]: to support rails 4

Comment (skyschub) [inline devise_browserid_authenticatable.gemspec]: So you'll kill every devise2 installation? And you'll update the gemspec if there is a .rc2 or a stable release? ;)

```
">= 2.1"
```

if you are really, really sure it is working with devise3.

**Comment by skyschub** [comment]
Heh. Thank you! I'll release the new version soon.

**Pull request merged**